dark/index.html

```diff
@@ -237,7 +237,7 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
           just below.
         </p>
         <p class="contact-text">
-          <i class="fa fa-envelope px-1"></i>Email (Personal) : yoshithasbrathnayake@gmail.com

```

Comment:
Don't remove this px-1 bcs it will remove the gap between that email and text @Rishav1707